### PR TITLE
Set up prod and dev envs

### PR DESCRIPTION
**Pull request by ibeckermayer**
Updated logo (hence zillions of binary diffs) and implemented the system @madhavajay designed and implemented in `covidwatch-ios-minimal` for managing the dev and prod schemes, which automatically injects your local ip into the `Info.plist`, making it trivial to talk to the Firestore emulator during development vs the live cloud functions in prod.

To test, throw a call to `getAppScheme()` in `actionsAfterLoading`, run the app via both the `covidwatch-ios-dev` and `covidwatch-ios-prod` schemes, and observe that the correct scheme name is printed in the console.

### CODE REVIEW

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Xcode Build
       run: |
         set -o pipefail
-        xcodebuild build -workspace COVIDWatch.xcworkspace -scheme 'COVIDWatch iOS' CODE_SIGNING_ALLOWED=NO | xcpretty
+        xcodebuild build -workspace COVIDWatch.xcworkspace -scheme 'covidwatch-ios-dev' CODE_SIGNING_ALLOWED=NO | xcpretty
```
Comment (jeffctown) [inline .github/workflows/pull_request.yml:21]: Should we add the prod one to the [fastfile](https://github.com/covid19risk/covidwatch-ios/blob/dev/fastlane/Fastfile)?

Comment (ibeckermayer) [reply .github/workflows/pull_request.yml:21]: Yes, good call.

**Review by madhavajay — APPROVED** [review]
Looks great, I just added a test environment so we can remove that old hardcoded test flag.

**Comment by madhavajay** [comment]
Weird, I rebased off dev so there should be no conflicts.

**Pull request merged**